OSDOCS-RHDEVDOCS-4304: shared resource updates for validating admission webhooks (#56497)
openshift/openshift-docs: modules/ephemeral-storage-csi-inline-overview.adoc

By default, {product-title} supports CSI inline ephemeral volumes with these limitations:

* Support is only available for CSI drivers. In-tree and FlexVolumes are not supported.

> * The Shared Resource CSI Driver supports inline ephemeral volumes as a Technology Preview feature, but only for the purpose of accessing `Secrets` or `ConfigMaps` across multiple namespaces.

Reviewer: ```suggestion
* The Shared Resource CSI Driver supports using inline ephemeral volumes to access `Secrets` or `ConfigMaps` across multiple namespaces as a Technology Preview feature.
```

Author: @rolfedh can we say `... inline ephemeral volumes only to access ...`

i.e. add `only`

Reviewer: > @rolfedh can we say `... inline ephemeral volumes only to access ...`
> 
> i.e. add `only`

Yes. that works.
